addons/web/static/src/views/widgets/ribbon/ribbon.scss

```diff
@@ -36,6 +36,10 @@
         &.o_medium {
             font-size: $font-size-base;
         }
+        &.o_above {
```

Comment:
we usually prefer to use bootstrap (in master: bootstrap 5.3) classes (when possible, obviously)
This allows to reduce our scss codebase and allows us to grasp the style of an element more readily